environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+
+kubectl get EdgePlacement
+NAME               AGE
+edge-placement-c   91s
+
+kubectl delete EdgePlacement edge-placement-c
+edgeplacement.edge.kcp.io "edge-placement-c" deleted
+```
+Placement translator logs:
+
+```bash
+:WorkspaceScheduled Status:True Severity: LastTransitionTime:2023-03-30 17:46:42 -0400 EDT Reason: Message:}] Initializers:[]}}
+I0330 17:47:01.732064   64918 main.go:119] "Receive" key="2vh6tnanyw60negt:edge-placement-c" val=map[{APIGroup: Resource:namespaces Name:commonstuff}:{APIVersion:v1 IncludeNamespaceObject:false}]
+I0330 17:47:01.732364   64918 main.go:119] "Receive" key="211ieqpc4xyydw2w:edge-placement-s" val=map[{APIGroup: Resource:namespaces Name:specialstuff}:{APIVersion:v1 IncludeNamespaceObject:false}]
+I0330 17:48:08.042551   64918 main.go:119] "Receive" key="2vh6tnanyw60negt:edge-placement-c" val=map[]
+```
+
+#### 4. Delete a kcp-edge Poc2023q1 example stage:
+
+```bash
+./delete_edge-mc.sh
+```
+
+## Bring your own workload (BYOW)
+
+#### 1. Create your own edge infrastructure (edge pclusters):
+
+For example: create a kind cluster
+
+```bash
+kind create cluster --name florin
+``` 
+
+#### 2. Deploy the kcp-edge platform:
+
+  * Step-1: Clone this repo:
+
+    ```bash
+    git clone https://github.com/kcp-dev/edge-mc
+    ```
+
+  * Step-2: change into the following directory path:
+
+    ```bash
+      cd edge-mc/environments/dev-env
+    ```
+
+  * Step-3: Deploy kcp-edge
+
+    ```bash
+    ./install_edge-mc.sh --stage 0
+    ```
+
+    This will start `kcp` and create/deploy the following components:
+
+    - 4 kcp workspaces: edge service provider workspace (`espw`), inventory management workspace (`imw`) and workload management workspace (`wmw`) under my-org workspace
+
+    ```bash
+    .
+    └── root
+        ├── compute
+        ├── espw
+        ├── imw-1
+        └── my-org
+            └── wmw-1
+    ```
+    - 3 kcp-edge controllers: [edge-scheduler](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/edge-scheduler/), [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) and [placement-translator](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/placement-translator/)
+
+    ```bash
+    ps aux | grep -e mailbox-controller -e placement-translator -e cmd/scheduler/main.go
+    user      2898   0.0  0.1 34922264  45188 s004  S     3:22PM   0:01.84 go run cmd/scheduler/main.go -v 2 --root-user shard-main-kcp-admin --root-cluster shard-main-root --sysadm-context shard-main-system:admin --sysadm-user shard-main-shard-admin
+    user      2872   0.0  0.2 34925136  56132 s004  S     3:22PM   0:02.44 go run ./cmd/mailbox-controller --inventory-context=shard-main-root -v=2
+    user      2929   0.0  0.2 34922964  69724 s004  S     3:22PM   0:03.74 go run ./cmd/placement-translator --allclusters-context shard-main-system:admin
+    ```
+
+#### 3. Deploy your own workload: 
+
+ * Populate the `imw`:
+    * Step-1: enter the target workspace:
```

Comment:
I think it would be better to speak here of the "inventory management workspace" rather than the "target workspace".